paper_trail_globalid.gemspec

```
@@ -26,8 +26,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec", "~> 3.3.0"
   spec.add_development_dependency "coveralls", "~> 0.8.2"
-  spec.add_development_dependency "activerecord", ">=3.2.0"
-  spec.add_development_dependency "activesupport", ">=3.2.0"
+  spec.add_development_dependency "activerecord", [">=3.2.0", "< 5.0"]
+  spec.add_development_dependency "activesupport", [">= 3.2.0", "< 5.0"]
```

also no need to limit the support to activeXXX since Rails 6 is about to be released
```suggestion
  spec.add_development_dependency "activerecord", ">=3.2.0"
  spec.add_development_dependency "activesupport",  ">= 3.2.0"
```